.github/workflows/job_cpu_functional_tests.yml

```diff
@@ -77,6 +70,7 @@ jobs:
           path: 'openvino'
 
       - name: Setup Python 3.11
+        if: ${{ runner.os != 'Linux' }} # We do not need to install Python as we use Docker on Linux with it installed
```

Comment:
is it possible to check within `./openvino/.github/actions/setup_python` that python is already installed and skip installation? to avoid conditions in workflow jobs